plugins/GSdx/GSTextureCache.h

```diff
@@ -152,7 +152,7 @@ class GSTextureCache
 		GSTexture* m_from_target;
 		GIFRegTEX0 m_layer_TEX0[7]; // Detect already loaded value
 		// Keep a GSTextureCache::m_map iterator to allow fast erase
-		std::array<uint16, MAX_PAGES> m_erase_it;
+		uint16 m_erase_it[MAX_PAGES];
```

Comment:
To uniform usage of arrays vs std::array in the class and also because the functionalities of std::array aren't used (bounds checking and static init).